server/src/main/resources/file.conf

```diff
@@ -118,6 +118,8 @@ transaction {
   #schedule delete expired undo_log in milliseconds
   undo.log.delete.period = 86400000
   undo.log.table = "undo_log"
+  #schedule check table meta
```

Comment:
This is a client configuration item.

Reply:
Thanks, I m changed

Reply:
> Thanks, I m changed

this file.conf located in server module, but server not use this configuration item.

Reply:
I move it to seata-config-core\src\main\resources\file.conf